atlp-rwanda/destructors-ec-fe
# #185335707 seller statistics

#### What does this PR do?
Seller statistics
#### Description of Task to be completed?
Seller will be able to see his/her statistics of a month
#### How should this be manually tested?
checkout to 
```
git checkout ft-seller-statistics-185335707
```
you will see an icon that directs you to dashboard
#### Screenshots (if appropriate)
![Screenshot (121)](https://github.com/atlp-rwanda/destructors-ec-fe/assets/50544115/51ce8287-ce26-4544-9268-e214bb11020a)

#### Questions:
